OWNERS

```diff
@@ -0,0 +1,23 @@
+
+filters:
+  ".*":
+    approvers:
+     - PK85 
+     - aszecowka 
+     - crabtree 
+     - pkosiec 
+     - kfurgol 
+     - tgorgol 
+     - dbadura
+     - akgalwas 
+     - janmedrek 
+     - Szymongib 
+     - franpog859 
+     - Maladie 
+  "README.md"
```

Comment:
Shouldn't we include all *.md files, not only READMEs? 😉

Reply:
Alright, I will modify it